packages/salesforce-adapter/src/filters/custom_objects.ts

```diff
@@ -14,7 +14,7 @@
 * limitations under the License.
 */
 import { logger } from '@salto-io/logging'
-import { collections, strings, multiIndex, promises } from '@salto-io/lowerdash'
+import { collections, strings, multiIndex, promises, values as valuesIsDefiend } from '@salto-io/lowerdash'
```

Comment:
I think the convention is calling this `lowerdashValues`
```suggestion
import { collections, strings, multiIndex, promises, values as lowerdashValues } from '@salto-io/lowerdash'
```